src/translations/en.json

```diff
@@ -399,7 +399,7 @@
           "show_entities": "Show entities"
         },
         "entity-attribute-picker": {
-          "attribute": "Attribute",
+          "attribute": "Attribute (optional)",
```

Comment:
HAving a field optional, should not be part of text or translations of the label of the field.

If that is needed, something is wrong with the UX. As the UI already knows if a field is optional or not.

IMHO this should not be merged, instead the issue should be fixed.

Reply:
🤷 Sorry didn't know... still think this should not be part of the label translations
The fact the label of the form field is (for example) "from", has nothing to do with it being optional or not.

Reply:
@piitaya / @frenck Let me know what you think of my latest commit. Also I haven't added a new translation before, so I'm not sure if I need to also add the key to the other languages. Note: I do not have a lokalise account, so I was not able to run `script/translations_download`. Please advise on whether this is needed to be run by me.

Reply:
According to the material guidelines:
<https://m3.material.io/components/text-fields/guidelines>
![CleanShot 2022-10-13 at 11 22 58](https://user-images.githubusercontent.com/5662298/195558011-4e5dba0e-baf9-4aed-b010-46830e5dfc0f.png)

Reply:
Since all fields here are optional except for the entity, we should not use the `optional` text here on any field

Reply:
Oh the material guidelines are strange... It looks inconsistent to have sometime asterisks, sometime optional labels. 🤔

Reply:
@travatomic Just editing translations file is enough 🙂

Reply:
I've changed this form to only indicate the required fields, not the optional fields. Updated screenshot:

![image](https://user-images.githubusercontent.com/934490/195631869-1d796ac3-1185-4131-b133-c3fb050fe7d3.png)